### DISCUSSION TITLE
Switching collector to Wear killed transmitter connection to phone

### DISCUSSION THREAD

**Discussion by Grntrenchman** [NightscoutFoundation/xDrip#2447] [General]
I have a oneplus 7t and ticwatch 3 pro. I've been using G6 native with xdrip+.
I wanted to see how the wear collector support was, so I enabled (and forced) it.
It worked ok, sync every 10 mins, but forced pairing every time (force unbond not enabled)
On disabling wear collector, ob1 fails with auth error.

Now, connecting to the transmitter always fails with auth error using ob1 collector no matter what. 
I've even tried a different (wiped) phone with fresh xdrip install, many different settings combos, old collector (ob1 disabled), nothing works. 
The transmitter *does* show in bluetooth list every 5 mins. 
I have 2 other transmitters, but 1 is very dead (anubis, battery removed), and the other is EOL but still ticking. neither have similar serial # (matching last 2).

The current transmitter will also no longer talk with Dexcom G6 official app (BYOD). searching for transmitter becomes transmitter not found after long time.

The wear collector would still sync after switching back to it from phone, but resetting xdrip data to try to fix stopped this from working. Now the transmitter won't sync at all with phone or watch.

I dragged out the hardware receiver, it is able to sync.

I've got the dead anubis here, so I'm rushing supplies to revive it, and switch to it.

I already sent logs through xdrip, let me know what else would help. 
After I switch to anubis, I'd be down for whatever troubleshooting we can come up with.

**Comment by Navid200** [comment]
If you are going to use a different transmitter, there is no troubleshooting to do.  

If you want to use the troubled transmitter, you have to completely disable the watch.  Changing xDrip settings is not a solution,.  It could be the cause of the problem.

 These are the required settings and steps:  https://navid200.github.io/xDrip/docs/Connectivity-troubleshoot

**Reply by Grntrenchman** [reply]
Uninstalled xdrip from watch, shut it off completely.
Force closed xdrip, cleared data, gave permissions, opened. Selected g6, entered transmitter serial and sensor code. No love. Not getting pairing request, sitting here watching log whole time.

![Screenshot_20221109-195413741](https://user-images.githubusercontent.com/29195872/200997348-024cb225-6dc2-4d6c-8aed-808249e97704.jpg)

**Reply by Navid200** [reply]
Can you post a screenshot of the system status page?
You can blackout the transmitter ID if you like.

Can you get a record of the battery voltages for the last 9 days as explained on the following link and post a screenshot?
https://navid200.github.io/xDrip/docs/Battery-condition

**Reply by Navid200** [reply]
You said the receiver is working, right?  Then, forget the battery voltages.

This is how you can trigger a pair request:
https://navid200.github.io/xDrip/docs/MissedPairRequest

**Reply by Grntrenchman** [reply]
![Screenshot_20221109-203328378](https://user-images.githubusercontent.com/29195872/201001379-11ad4abe-927c-4c4c-a6da-a9597c2313a5.jpg)
This is after retriggering pairing by changing sn away and back.

**Reply by Grntrenchman** [reply]
Followed by:
![Screenshot_20221109-203530184](https://user-images.githubusercontent.com/29195872/201001614-6edaf7f9-2748-41d3-8df7-1c0351c3f3d7.jpg)

**Reply by Grntrenchman** [reply]
It's happily chugging along with the hardware one though.
![IMG_20221109_203300](https://user-images.githubusercontent.com/29195872/201001751-33bb3b8f-8d02-433f-8c3a-dfd81c2098a1.jpg)

**Reply by Grntrenchman** [reply]
I haven't been getting any form of pairing request or notification FYI.

**Comment by Grntrenchman** [comment]
![Screenshot_20221109-204455238](https://user-images.githubusercontent.com/29195872/201002739-fcd35b27-a99d-4be0-a9c0-66b2d54a7b3d.jpg)
![Screenshot_20221109-204445129](https://user-images.githubusercontent.com/29195872/201002745-59c129c4-3155-4249-9ac5-cdbd0b0d3e1c.jpg)
![Screenshot_20221109-204434914](https://user-images.githubusercontent.com/29195872/201002759-c1368b2b-5d69-464d-8420-56969695ba8c.jpg)
![Screenshot_20221109-204352972](https://user-images.githubusercontent.com/29195872/201002769-2b9852ae-7b58-47a1-aa12-46e213cadc0e.jpg)
Shouldn't be a permissions or allowance issue...

**Reply by Navid200** [reply]
Is there anything else that could be using the phone slot?
Do you have the Dexcom app installed on another device?
Do you have xDrip paired with the same transmitter on another phone?
Do you have a BlueJay watch?

You obviously did not follow this:  https://navid200.github.io/xDrip/docs/MissedPairRequest
Any reason?

**Reply by Navid200** [reply]
Or, did you?
If you did, do you have old phones at home you can try for an experiment?

**Reply by Navid200** [reply]
They can be as old as Android 5.

**Reply by Grntrenchman** [reply]
I did, this was after. 
Wife's phone is a follower, no direct listening. 
I wiped the old phone just now, but it's been off. 
Watch is still in "essential mode", which is shut down with a minimal os for driving the FSTN display.

I don't have anything else other than the official hardware receiver, which is still hooked up and working.

**Reply by Grntrenchman** [reply]
![Screenshot_20221109-205511637](https://user-images.githubusercontent.com/29195872/201004939-67dba32a-51d2-46f9-a8ba-326bccfb454a.jpg)

**Reply by Grntrenchman** [reply]
Then
![Screenshot_20221109-205845416](https://user-images.githubusercontent.com/29195872/201004981-41a17267-4c60-4394-b4b3-025a1e0d75b5.jpg)

**Reply by Navid200** [reply]
Right.
Do you have old phones or can you borrow one from a friend?

**Reply by Grntrenchman** [reply]
I have the oneplus 3t I just wiped. Want me to try something else? I can get xdrip on it fairly fast if it will work.

**Reply by Navid200** [reply]
What do you mean by the old phone?
I thought you only had one phone and one watch.

**Reply by Grntrenchman** [reply]
I have oneplus 7t that is my current phone.
I've tried connecting with oneplus 3t (old phone) as mentioned in the first post to rule out phone issue.

**Reply by Grntrenchman** [reply]
I can probably provide you direct ADB access to the 3t with xdrip on it if it would be of any use.

**Reply by Navid200** [reply]
Is there a third phone you can use?
If yes, disable collection on the other two phones.
Then, set up this third phone and hopefully, it will connect.
After it does, wait for it to show the battery voltages.  It may take up to half an hour.
After the battery voltages are shown, disable collection, and set it up on the old phone.  It should then connect.  Again, wait until the battery voltages show.
Then, disable collection on that one and set up your new phone and it should connect.

**Reply by Grntrenchman** [reply]
I'll try to borrow one from a roommate. I'll disable xdrip on the main phone, #2 is still wiped lol. Report back in 10ish probably.

**Reply by Navid200** [reply]
@joshbohmer

**Reply by Navid200** [reply]
You don't have BYOD active, do you?

**Reply by Grntrenchman** [reply]
I switched from BYOD to xDrip this transmitter. it's never been connected. it's on the phone, but disabled. I can remove it completely.

**Reply by Grntrenchman** [reply]
same deal on roommate's phone, a Samsung N920A (Note 4?).
finds transmitter, no notification for pairing, goes to auth error.

**Reply by Navid200** [reply]
Pair notification only comes up on Android 10 and higher.

**Reply by Navid200** [reply]
Can you show what the system status page show on this third phone?

**Reply by Navid200** [reply]
Considering this is a Samsung, you need to disable trust auto-connect.
These are the settings:  https://navid200.github.io/xDrip/docs/G6-Recommended-Settings

**Reply by Navid200** [reply]
If you see the same error on the third phone and if your settings are all OK, I'm not sure what's going on.

However, if your settings are not correct, I' wouldn't be surprised.
I need to go to bed now.

**Reply by Grntrenchman** [reply]
Screenshot combo doesn't work on their phone, buttons are... worn. not in power list either. 
It came out same, "connected" but failed to auth after several tries.
I've returned it to them, and dug out old android box. Now armed with 3x Oneplus 3t, one stock android 9 (the one I was testing earlier), other two are coltOS android 10.

**Reply by Grntrenchman** [reply]
No worries. Not much you can debug since the transmitter is a black box. Something went wrong on handoff to or from wear...

existing plan goes forward, ditch this transmitter once I can get anubis working.

**Reply by Navid200** [reply]
Temporarily, turn off the receiver.
Make sure the settings are all correct.  Does it connect then?

**Reply by Grntrenchman** [reply]
Still no love. Always fun to find a new failure method. I'll try to trigger it with the anubis when the battery arrives, before I close it up, and see if it's repeatable.
Otherwise, since it won't talk to the BYOD app either, I'll probably contact Dexcom next and see if they want it to exchange it for dissection.

**Reply by Navid200** [reply]
But, it works with your receiver, right?
Did you turn off the receiver for a try?  Did it not connect even then?

**Reply by Grntrenchman** [reply]
It does work with the receiver. I turned it, the original phone, and watch off completely. It did not connect to a 3t with fresh xdrip install, giving same auth errors.

While it "works", this receiver doesn't have prediction, proper alerts, or even a visible delta, I'm switching away from it ASAP once the Anubis is operational.

**Reply by Navid200** [reply]
Do you want to try an experiment?
Have a look at this first:  https://navid200.github.io/xDrip/docs/G6_slot

Temporarily turn of the receiver, switch the slot in xDrip and see if it connects.

**Reply by Grntrenchman** [reply]
Trying now, slot 1 selected... It shows connecting, auth request, no connection. Still auth error. Definitely the transmitter.. something in the OB1 -> wear transfer messed up the handshake good. Again, even Dexcom app won't see the thing.

**Reply by theunknown86** [reply]
> Trying now, slot 1 selected... It shows connecting, auth request, no connection. Still auth error. Definitely the transmitter.. something in the OB1 -> wear transfer messed up the handshake good. Again, even Dexcom app won't see the thing.

Run the AnubisT00L tool on it and post the screenshot. Make sure nothing else is talking first (xDrip, Wear, Receiver etc) and delete the existing BT pairing. Doesn't matter that it's stock.

Also, you don't happen to have another transmitter in the house with the same last 2 characters? I've seen that cause Auth Errors before.

**Reply by Grntrenchman** [reply]
@joshbohmer ok, I'll try AnubisT00L. I don't have a matching transmitter. You actually helped me recently run this down trying to connect to the dead anubis (had a transmitter matching *that*). This is the one I started once it was decided the Anubis I have on hand was dead dead.

Only 3 transmitters here, Anubis ending in AE, old transmitter EOL ending in AE, and this one.

**Reply by Grntrenchman** [reply]
It eventually asked to pair, and here we are.
Should I try to connect in xdrip or G6 now that it's "paired"?

![Screenshot_20221110-133840232](https://user-images.githubusercontent.com/29195872/201211364-5bcd327c-45de-4044-bde3-9db3db4fdb37.jpg)

**Reply by theunknown86** [reply]
> It eventually asked to pair, and here we are. Should I try to connect in xdrip or G6 now that it's "paired"?

Yep, give it a go with xDrip set to slot 2 now

**Reply by Grntrenchman** [reply]
> Yep, give it a go with xDrip set to slot 2 now

Ok, reenabled xdrip, set to g6, changed to slot 2, changed serial to and back. Waiting for connection.

Just "Connected, and failed with same "Checking Auth", then error.
I'll post SS of logs etc.

**Reply by Grntrenchman** [reply]
![Screenshot_20221110-135032917](https://user-images.githubusercontent.com/29195872/201213269-d136f38e-0625-44c3-9356-cc529390591e.jpg)
![Screenshot_20221110-135029889](https://user-images.githubusercontent.com/29195872/201213276-2978ec3c-5afe-4044-bf98-89422a472042.jpg)
![Screenshot_20221110-135025080](https://user-images.githubusercontent.com/29195872/201213292-05d4e377-ee83-4ada-99ab-3a832fe84d44.jpg)
![Screenshot_20221110-135015165](https://user-images.githubusercontent.com/29195872/201213307-42cbd16a-cd6e-4d8c-91f7-dec69558d903.jpg)
![Screenshot_20221110-135007546](https://user-images.githubusercontent.com/29195872/201213326-65ce08b5-29df-4420-98f7-8836b04b0be0.jpg)
![Screenshot_20221110-135000466](https://user-images.githubusercontent.com/29195872/201213338-d6d6b5c2-e9e6-45b5-a000-398ea5ff6238.jpg)
![Screenshot_20221110-134949575](https://user-images.githubusercontent.com/29195872/201213348-82c9a3a0-07e2-4341-a198-94f03f2c2287.jpg)
![Screenshot_20221110-134909588](https://user-images.githubusercontent.com/29195872/201213359-59985490-7b56-46c5-9ac6-51de5162ba81.jpg)

**Reply by Navid200** [reply]
Allow OB1 unbonding should not be enabled.
But, that's not going to solve the current problem.

**Reply by Grntrenchman** [reply]
I've received supplies to get the Anubis working. I'll try to recreate the issue there before I seal it up.

**Comment by rudysroost** [comment]


**Reply by Grntrenchman** [reply]
This isn't similar, I can't get transmitter to connect to any phone or app. Start a new discussion if you want help, posting like this will just muddle things with your different issue.

**Comment by Grntrenchman** [comment]
I've received supplies to get the Anubis working. I'll try to recreate the issue there before I seal it up.

**Reply by Grntrenchman** [reply]
Swapsied back and forth between wear collector and OB1 for a few hours, Couldn't get this issue to recur.
